readme.md

```diff
@@ -1,404 +1,61 @@
 # AngularFire
-- Create and configure a Firebase back-end
-- Create a Javascript that communicates with a Firebase back-end
-- Explain the difference between HTTP and Websockets, and provide an advantage and disadvantage of each
-- Explain what AngularFire is and how it differs from Firebase
-- Deploy your app to Firebase Hosting
+- Create and configure a Firebase back-end.
+- Create a Javascript that communicates with a Firebase back-end.
+- Explain the difference between HTTP and Websockets, and provide an advantage and disadvantage of each.
+- Explain what AngularFire is and how it differs from Firebase.
+- Deploy your app to Firebase Hosting.
 
+## Framing
 
-## Introduction (10 min)
+Up until this point, we've used Postgres and SQL to persist data. Today we'll be exploring an alternative: Firebase.
 
-You've heard that Postgresql is not our only database option.  Today, we'll investigate one, Firebase. Firebase is a Platform as a Service (PaaS) that provides a graphical interface to set up a back end, both the database and api.
+## Firebase
 
-We will be using the AngularFire library to access Firebase.
+#### What Is Firebase?
 
-Let's learn a little about both by skimming the [AngularFire Intro](https://www.firebase.com/docs/web/libraries/angular/guide/intro-to-angularfire.html).  Unfortunately, they will not allow all of us to sign up for free accounts because we are all sharing the same IP address and they have limits.  If you have an account feel free to create your own Firebase and use it during this lesson.
+Firebase is a cloud-based database service. Not only can we access it programmatically using code, but we can also interact with data via a graphical interface in the browser.
 
```

Comment:
Firebase is technically a PaaS - platform as a service. Today we are going to be using it to add a cloud-based database to our front-end applications.